guides/navplace.md

```diff
@@ -0,0 +1,298 @@
+---
+title: How to Use the navPlace Extension With Leaflet
+id: using_navplace
+layout: spec
+author: Bryan Haberberger
+breadcrumbs:
+ - label: Get started
+   link: '{{ site.root_url }}/get-started'
+ - label: Guides
+   link: '/'
+redirect_from:
+  '{{ site.guides_url }}/guides/iiif.io.api.extension.navplace/'
+
+---
+
+The IIIF Presentation API 3 was extended and now includes a property called `navPlace`. This property contains geographic coordinates in the form of [GeoJSON-LD](https://geojson.org/geojson-ld/). This guide will show `navPlace` within a IIIF Manifest, and then demonstrate how to process and display the information from the property using javascript and the Leaflet JS library.  
+
+## Why Leaflet?
+Leaflet is one of the most widely used and easy to understand javascript web mapping libraries. With excellent API documentation, tutorial materials and broad browser compatibility, using Leaflet as the base will offer the most understandable instructions to a large audience. You can learn more about Leaflet at [https://leafletjs.com/](https://leafletjs.com/).
```

Comment:
Suggested changes:

Leaflet is one of the most widely used and thoughtfully designed javascript web mapping libraries. With excellent API documentation, tutorial materials and broad browser compatibility, Leaflet offers easy to follow instructions to a large audience. You can learn more about Leaflet at [https://leafletjs.com/](https://leafletjs.com/).